doc.go

```diff
@@ -400,6 +400,15 @@ The “?“ inside the “VALUES“ clause specifies that the SQL statement uses
 Binding data that involves time zones can require special handling. For details, see the section
 titled "Timestamps with Time Zones".
 
+The driver takes advantage of sql.Null types which enables the proper handling of null parameters inside function calls, i.e.:
```

Comment:
Should we provide the information from which driver version the feature is available?

Reply:
good idea, would be very useful and help avoid confusion